proto/participation/query.proto

```diff
@@ -15,7 +17,24 @@ service Query {
   rpc Params(QueryParamsRequest) returns (QueryParamsResponse) {
     option (google.api.http).get = "/tendermint/spn/participation/params";
   }
-  // this line is used by starport scaffolding # 2
+  // Queries a UsedAllocations by address.
+	rpc UsedAllocations(QueryGetUsedAllocationsRequest) returns (QueryGetUsedAllocationsResponse) {
+		option (google.api.http).get = "/tendermint/spn/participation/used_allocations/{address}";
+	}
+	// Queries a list of UsedAllocations items.
+	rpc UsedAllocationsAll(QueryAllUsedAllocationsRequest) returns (QueryAllUsedAllocationsResponse) {
+		option (google.api.http).get = "/tendermint/spn/participation/used_allocations";
+	}
+	// Queries a AuctionUsedAllocations by address and auctionID.
+	rpc AuctionUsedAllocations(QueryGetAuctionUsedAllocationsRequest) returns (QueryGetAuctionUsedAllocationsResponse) {
+		option (google.api.http).get = "/tendermint/spn/participation/auction_used_allocations/{address}/{auctionID}";
+	}
+	// Queries a list of AuctionUsedAllocations items.
```

Comment:
We should have a query to get all `AuctionUsedAllocations` for a specific address to get all the allocation/auction association for a specific address

I would consider instead of creating a new query to set it for `AuctionUsedAllocationsAll`, so adding  the `address` param to the query and fetching all the `AuctionUsedAllocations` for this address.
And don't think having this query to get all `AuctionUsedAllocations` for all addresses would be necessary. All allocations used by addresses can be queried with `UsedAllocationsAll`, then the user could query `AuctionUsedAllocationsAll` with a specific address to get more details